docs/about/usage-tracking.md

```diff
@@ -0,0 +1,51 @@
++++
+title = "Usage Tracking"
+description = "Learn about usage tracking for Kubeflow on AWS"
+weight = 50
++++
+
+AWS uses customer feedback and usage information to improve the quality of the services and software we offer to customers. We have added usage data collection to the AWS Kubeflow distribution in order to better understand customer usage and guide future improvements. Usage tracking for Kubeflow is activated by default, but is entirely voluntary and can be deactivated at any time. 
+
+Usage tracking for Kubeflow on AWS collects the instance ID used by one of the worker nodes in a customer’s cluster. This data is sent back to AWS once per day. Usage tracking only collects the EC2 instance ID where Kubeflow is running and does not collect or export any other data to AWS. If you wish to deactivate this tracking, instructions are below. 
+
+### Activate usage tracking
+
+Usage tracking is activated by default. If you deactivated usage tracking for your Kubeflow deployment and would like to activate it after the fact, you can do so at any time with the following command:
+
+```
+kustomize build distributions/aws/aws-telemetry | kubectl apply -f -
```

Comment:
as pointed out by Suraj 

Update this line to 
`kustomize build awsconfigs/common/aws-telemetry | kubectl apply -f -`

Reply:
Did that in my latest push: https://github.com/awslabs/kubeflow-manifests/pull/188/commits/a15d5bd5fb7ed95138c5997602aa6b5f5bfbff90#diff-a693f8b2d8b72223fa6a82d2d8273df89eb8db076fe2865fbae51d14fcb4e3df